x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/index.tsx

```diff
@@ -27,15 +25,13 @@ import { SearchIndicesRouter } from './components/search_indices';
 import {
   CONNECTORS_PATH,
   CRAWLERS_PATH,
-  ERROR_STATE_PATH,
   ROOT_PATH,
   SEARCH_INDICES_PATH,
   SETUP_GUIDE_PATH,
 } from './routes';
 
 export const EnterpriseSearchContent: React.FC<InitialAppData> = (props) => {
   const { config } = useValues(KibanaLogic);
```

Comment:
can we also clean up `config` from ent-search references?

Reply:
Done merging changes from previous PRs